pom.xml

```diff
@@ -1336,6 +1337,12 @@
         <version>${junit.version}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>pl.pragmatists</groupId>
```

Comment:
Where are you using this dependency?

Reply:
This test dependency is used for running Paramaterized unit tests.

Reply:
The junit library already supports parameterized test. Should need this extra dep.

e.g. https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/runtime/distributed/ProgramTwillRunnableModuleTest.java